ProjectName/jest.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
I see we have `jest` configured inside `package.json`.
Please let's have a single place where we do this. I would suggest removing it from the `package.json`, if you aim to add a separate file.

Reply:
Agree with what Oleks said, it is always good to follow YAGNI Principle, to implement things when you actually need them, never when you just foresee that you need them.